src/VirtoCommerce.CatalogModule.Web/module.manifest

```diff
@@ -3,7 +3,7 @@
   <id>VirtoCommerce.Catalog</id>
   <version>3.63.0</version>
   <version-tag />
-  <platformVersion>3.68.0</platformVersion>
+  <platformVersion>3.78.0</platformVersion>

```

Comment:
Update NuGet dependencies. For example https://github.com/VirtoCommerce/vc-module-store/blob/dev/src/VirtoCommerce.StoreModule.Data/VirtoCommerce.StoreModule.Data.csproj#L30

Reply:
OK